c2corg_api/views/forum.py

```diff
@@ -86,12 +97,25 @@ def __init__(self, request):
     def collection_post(self):
         settings = self.request.registry.settings
 
+        import pprint
         locale = self.request.validated['locale']
+        document = self.request.validated['document']
+        log.warning(pprint.pformat(dir(locale)))
```

Comment:
Is warning the good level? Maybe debug ?

Reply:
This was debug code I removed in 7bf9533